src/mito2/src/memtable/key_values.rs

```diff
@@ -65,6 +65,52 @@ impl KeyValues {
     }
 }
 
+/// Key value view of a mutation.
+#[derive(Debug)]
+pub struct KeyValuesRef<'a> {
+    /// Mutation to read.
+    ///
+    /// This mutation must be a valid mutation and rows in the mutation
+    /// must not be `None`.
+    mutation: &'a Mutation,
+    /// Key value read helper.
+    helper: SparseReadRowHelper,
+}
+
+impl<'a> KeyValuesRef<'a> {
+    /// Creates [crate::memtable::KeyValues] from specific `mutation`.
+    ///
+    /// Returns `None` if `rows` of the `mutation` is `None`.
+    pub fn new(metadata: &RegionMetadata, mutation: &'a Mutation) -> Option<KeyValuesRef<'a>> {
```

Comment:
> It seems it always returns `Some()`

Just in case `mutation.rows` is None: https://github.com/GreptimeTeam/greptimedb/blob/b9a2a0d186dd8e14f94bbae6f849e8d50a820ddb/src/mito2/src/memtable/key_values.rs#L85